pages/api/Schema/reports/designationAccounts/designationAccounts.graphql

```diff
@@ -0,0 +1,31 @@
+extend type Query {
+  designationAccounts(accountListId: ID!): [DesignationAccountsGroup!]!
+}
+
+extend type Mutation {
+  setActiveDesignationAccount(
+    accountListId: ID!
+    designationAccountId: ID!
+    active: Boolean!
```

Comment:
I see you added this. But I think you actually need to pass it on to the PUT request.

Reply:
Thanks for remind me this.